### PR TITLE
titty milk & cum

### PR DESCRIPTION
**Pull request by Pokebutt**
## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

**Comment by chemistrymain2** [comment]
No.

File: code/modules/arousal/arousal.dm [hunk 1]
```diff
@@ -223,8 +223,41 @@
...
+
+
+
 	mb_cd_timer = world.time + mb_cd_length
+
+/mob/living/carbon/human/verb/climax_verb()
+	set category = "IC"
+	set name = "Climax"
+	set desc = "Lets you choose a couple ways to ejaculate."
+	mob_climax()
```
Comment (Kitsunemitsu) [inline code/modules/arousal/arousal.dm:263]: Need trailing endline

File: code/modules/arousal/genitals.dm [hunk 2]
```diff
@@ -348,4 +348,4 @@
 			dna.features["breasts_color"] = dna.features["cock_color"]
 		else if (willyCheck == FALSE)
 			dna.features["cock_color"] = dna.features["breasts_color"]
-	return TRUE
+	return TRUE
```
Comment (Kitsunemitsu) [inline code/modules/arousal/genitals.dm:351]: Need trailing endline.

File: code/modules/arousal/organs/breasts.dm [hunk 3]
```diff
@@ -17,8 +17,8 @@
 	arousal_verb = "Your breasts start feeling sensitive"
 	unarousal_verb = "Your breasts no longer feel sensitive"
 	orgasm_verb = "leaking"
-	fluid_transfer_factor = 0.5
-	var/static/list/breast_values = list("a" =  1, "b" = 2, "c" = 3, "d" = 4, "e" = 5, "f" = 6, "g" = 7, "h" = 8, "i" = 9, "j" = 10, "k" = 11, "l" = 12, "m" = 13, "n" = 14, "o" = 15, "huge" = 16, "flat" = 0)
+	fluid_transfer_factor = 1
+	var/static/list/breast_values = list("a" =  2, "b" = 4, "c" = 6, "d" = 8, "e" = 10, "f" = 6, "g" = 7, "h" = 8, "i" = 9, "j" = 10, "k" = 11, "l" = 12, "m" = 13, "n" = 14, "o" = 15, "huge" = 16, "flat" = 0)
```
Comment (Kitsunemitsu) [inline code/modules/arousal/organs/breasts.dm:21]: Why does E sized breasts give more milk than I sized breasts

File: code/modules/arousal/organs/breasts.dm [hunk 4]
```diff
@@ -125,4 +123,4 @@
 	toggle_visibility(D.features["breasts_visibility"], FALSE)
 
 #undef BREASTS_ICON_MIN_SIZE
-#undef BREASTS_ICON_MAX_SIZE
+#undef BREASTS_ICON_MAX_SIZE
```
Comment (Kitsunemitsu) [inline code/modules/arousal/organs/breasts.dm:126]: Needs trailing endline

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Please check these.

**Comment by Kitsunemitsu** [comment]
Also, please check your changelog and PR info.
You should ALWAYS check these

File: code/modules/arousal/organs/breasts.dm (hunk 3 above)
Comment (Pokebutt) [reply code/modules/arousal/organs/breasts.dm:21]: E is the actual cap

**Comment by Pokebutt** [comment]
> No.

frok stop being a cockblock

**Comment by biotear** [comment]
> lock

Agreed. Add the cum back. It's funny.

**Comment by Legate-Maxson** [comment]
This code doesnt function and it doesnt compile.

**Comment by ogrigbe** [comment]
What raptor said, it literally doesn't work.

**Comment by Pokebutt** [comment]
> This code doesnt function and it doesnt compile.

lemme try again

**Comment by Pokebutt** [comment]
> What raptor said, it literally doesn't work.

works fine for me

**Pull request closed without merging**